app/views/dogs/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Dogs#create</h1>
+<p>Find me in app/views/dogs/create.html.erb</p>
```

Comment:
Shouldn't it be new for the view, rather than create? I  thought new was the form and create is only for the POST action.

Reply:
I imagined that on the user show page we would have a button to add a dog and when you press this button a form pops up, still in the show, where you can create the dog profile without having to load a "new" page

Reply:
okay i see !